Gemfile

```diff
@@ -1,5 +1,7 @@
 source 'https://rubygems.org'
 
+ruby '2.2.1'
+
 #========== GLOBAL GEMS ======================
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '4.2.5'
```

Comment:
ok, i would like to upgrade to latest stable release. Is there a tool to detect stale code and remind me what is not up to date and point me to release notes? As i recall, under Unbuntu linux you can always get a preview list of what updates are available for Ubuntu, for example. I am going to merge and then upgrade.